benches/benches/block_target_gas_set/memory.rs

```diff
@@ -28,12 +28,7 @@ pub fn run_memory(group: &mut BenchmarkGroup<WallTime>) {
     run(
         "memory/aloc opcode",
         group,
-        [
-            op::movi(0x10, 1000),
-            op::aloc(0x10),
-            op::jmpb(RegId::ZERO, 0),
-        ]
-        .to_vec(),
+        [op::movi(0x10, 0), op::aloc(0x10), op::jmpb(RegId::ZERO, 0)].to_vec(),
```

Comment:
What's the difference between `0` and `1000` here?

Reply:
If you allocate `1000` you can run out of memory faster than run out of gas=) Because it doesn't make any difference to allocate `0` or `1000` for stability of the check, it is better to allocate zero.